deploy/crds/k8s_v1alpha1_iofog_crd.yaml

```diff
@@ -10,10 +10,105 @@ spec:
     plural: iofogs
     singular: iofog
   scope: Namespaced
+  validation:
+    openAPIV3Schema:
+      properties:
+        apiVersion:
+          description: 'APIVersion defines the versioned schema of this representation
+            of an object. Servers should convert recognized schemas to the latest
+            internal value, and may reject unrecognized values. More info: https://git.k8s.io/community/contributors/devel/api-conventions.md#resources'
+          type: string
+        kind:
+          description: 'Kind is a string value representing the REST resource this
+            object represents. Servers may infer this from the endpoint the client
+            submits requests to. Cannot be updated. In CamelCase. More info: https://git.k8s.io/community/contributors/devel/api-conventions.md#types-kinds'
+          type: string
+        metadata:
+          type: object
+        spec:
+          properties:
+            microservices:
+              items:
+                properties:
```

Comment:
Don't we need volume mappings?

Reply:
This is auto generated using operator sdk. I haven't changed the code that would change this auto generated yaml

Reply:
this is commented out - is this intentional?
#      volume-mappings:
#        - hostDestination: "/etc/iofog-agent"
#          containerDestination: "test-mapping"
#          accessMode: "rw"